src/main/resources/org/archive/commons.properties

```diff
@@ -0,0 +1,5 @@
+operator=<enter operator name here>
+publisher=<enter publisher name here>
+wat.warcinfo.description=<enter warc info description here>
```

Comment:
My apologies, I think I forgot to push my commit, and I can't access my desktop right now, I'll do it tomorrow morning.